sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2964,14 +2972,6 @@ object SQLConf {
       .booleanConf
       .createWithDefault(false)
 
-  val ENFORCE_RESERVED_KEYWORDS = buildConf("spark.sql.ansi.enforceReservedKeywords")
```

Comment:
Note related to this PR, but I move this code next to the ANSI_ENABLED flag for grouping the ANSI SQL related flags.